test/Microsoft.AspNet.Mvc.Core.Test/DefaultActionDescriptorCollectionProviderLoggingTest.cs

```diff
@@ -72,7 +73,7 @@ public void ControllerDiscovery()
             Assert.Empty(controllerModelValues.AttributeRoutes);
             Assert.Empty(controllerModelValues.RouteConstraints);
             Assert.NotEmpty(controllerModelValues.Attributes);
-            Assert.Single(controllerModelValues.Filters);
+            Assert.Empty(controllerModelValues.Filters);
```

Comment:
This shouldn't have changed - fix the `ControllerModelBuilder` and this will be the same as before